modules/bound-sa-tokens-configuring.adoc

```diff
@@ -50,6 +42,46 @@ spec:
 ----
 <1> This value should be a URL from which the recipient of a bound token can source the public keys necessary to verify the signature of the token. The default is [x-]`https://kubernetes.default.svc`.
 
+.. Save the file to apply the changes.
+
+.. Optional: Manually restart all pods in the cluster so that the holder will request a new bound token.
+
+... Wait for a new revision of the Kubernetes API server pods to roll out. It can take several minutes for all nodes to update to the new revision. Run the following command:
++
+[source,terminal]
+----
+$ oc get kubeapiserver -o=jsonpath='{range .items[0].status.conditions[?(@.type=="NodeInstallerProgressing")]}{.reason}{"\n"}{.message}{"\n"}'
+----
++
+Review the `NodeInstallerProgressing` status condition for the Kubernetes API server to verify that all nodes are at the latest revision. The output shows `AllNodesAtLatestRevision` upon successful update:
++
+[source,terminal]
+----
+AllNodesAtLatestRevision
+3 nodes are at revision 12 <1>
+----
+<1> In this example, the latest revision number is `12`.
++
+If the output shows a message similar to one of the following messages, the update is still in progress. Wait a few minutes and try again.
+
+** `3 nodes are at revision 11; 0 nodes have achieved new revision 12`
+** `2 nodes are at revision 11; 1 nodes are at revision 12`
```

Comment:
we should not rely on any specific messages, they are not an API and can be changed at any moment. I wonder whether there's a better way to describe the wait.

Reply:
@stlaz I get what you mean, but we do use this method of checking a few other places in the docs now. How about we move forward with this for now, and then can re-evaluate if we have a better way to check that we can update in all places.

Reply:
Sure. I am just hoping that noone will ever configure their tooling to be dependent on this very message as that would be wrong.